core/trino-main/src/test/java/io/trino/metadata/TestInformationSchemaMetadata.java

```diff
@@ -38,6 +38,8 @@
 import io.trino.testing.LocalQueryRunner;
 import io.trino.transaction.TransactionId;
 import io.trino.transaction.TransactionManager;
```

Comment:
Same as you did already on  d57d6602 it would be clearer to some why doing the resource allocation within the test constructor is a bad idea.

> Maven surefire plugin does not report full exception when it happens during test class constructor call.

Reply:
I am automating checks now: https://github.com/trinodb/trino/pull/15165
So we can improve comments / error reporting in ReportResourceHungryTests in that PR.